src/playlist-loader.js

```diff
@@ -274,6 +274,34 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
     return (loader.master.playlists.filter(isEnabled).length === 1);
   };
 
+  /**
+   * Returns whether the current playlist is an AES encrypted HLS stream
+   *
+   * @return {Boolean} true if it's an AES encrypted HLS stream
+   */
+  loader.isAes_ = function(media) {
```

Comment:
I think these would be better suited as utility functions in`playlist.js`. They don't rely on any state of the loader so lets not put them on it